PROJECT_CHARTER.md

```diff
@@ -67,7 +67,35 @@ Section Intentionally Left Blank
 
 ### Section 4.2: Decision-making, Voting, and/or Elections
 
-Section Intentionally Left Blank
+**Decision-making**
+
+The Fastify's features can be discussed in the GitHub's issues and/or projects.
+The consensus on a discussion is reached when:
+
+- the majority of the Collaborator agree on the decision
+- there is not any blocking concern by the Lead Maintainers
+
+Whenever there is not consensus, Lead Maintainers will have final say on the topic.
+
+**Voting, and/or Elections**
+
+Individuals making significant and valuable contributions to the project may be candidate to join
+the Fastify's org.
+
+A Fastify Team Member needs to open a private team discussion on GitHub and list the candidates who
+wants to sponsor with a link to the user's contributions (eg: `[USERNAME](https://github.com/search?q=author:USERNAME+org:fastify)`).
+
+Otherwise, a Contributor may self-apply if he/she thinks to meet the above criteria reaching out
+to a Lead Maintainer privately with a few links to his/her valuable contributions.
+The Lead Maintainer will reply to the Contributor and will decide if candidate he/she to the FTM.
```

Comment:
```suggestion
Otherwise, a Contributor may self-apply if they believe they meet the above criteria by reaching out
to a Lead Maintainer privately with a few links to their valuable contributions.
The Lead Maintainer will reply to the Contributor and will decide if candidate he/she to the FTM.
```

I don't know what this is supposed to mean:

>  will decide if candidate he/she to the FTM.

What is "FTM"?

Reply:
At the beginning of the file it is specified:

> Technical leadership for the projects within the [OpenJS Foundation][openjs foundation] is delegated to the projects through their project charters by the [OpenJS Foundation Cross-Project Council](https://openjsf.org/about/governance/) (CPC). In the case of the Fastify project, it is delegated to the [Fastify's Team](README.md#team) (FTM).